### DISCUSSION TITLE
刷抖音链接暴增问题解决办法 Also force_ttl_rules.txt

### DISCUSSION THREAD

**Discussion by xiebaiyuan** [kkkgo/PaoPaoDNS#166] [经验分享]
修改custom_mod.yaml 把douyin这堆cdn直接扔alidns或者抖音自己的dns就好了

 - zone: smtcdns.net
   dns: udp://223.5.5.5:53,udp://223.6.6.6:53
   ttl: 0
   seq: top
   socks5: no
 - zone: weilayun.com
   dns: udp://223.5.5.5:53,udp://223.6.6.6:53
   ttl: 0
   seq: top
   socks5: no
 - zone: idouyinvod.com
   dns: udp://223.5.5.5:53,udp://223.6.6.6:53
   ttl: 0
   seq: top
   socks5: no

----

或者 ：
update my force_ttl_rules.txt and RULES_TTL = 10 （magic）
douyinpic.com@180.184.1.1:53
douyincdn.com@180.184.1.1:53
amemv.com@180.184.1.1:53
douyinvod.com@180.184.1.1:53
douyinliving.com@180.184.1.1:53
douyinstatic.com@180.184.1.1:53
douyin.com@180.184.1.1:53
idouyinvod.com@180.184.1.1:53
byteimg.com@180.184.1.1:53
bytegecko.com@180.184.1.1:53

**Comment by xiebaiyuan** [comment]
也可以把ali的换成douyin自己的 180.184.1.1  180.184.2.2

**Comment by Daohuai0913** [comment]
请问这种cdn在哪可以找到啊？

**Reply by xiebaiyuan** [reply]
套娃一个adguardhome然后只刷抖音，就会发现一大堆cdn飘过。 目前加了这些。 应该还有一些

- zone: smtcdns.net
   dns: udp://223.5.5.5:53,udp://180.184.1.1:53
   ttl: 0
   seq: top
   socks5: no
 - zone: weilayun.com
   dns: udp://223.5.5.5:53,udp://180.184.1.1:53
   ttl: 0
   seq: top
   socks5: no
 - zone: idouyinvod.com
   dns: udp://223.5.5.5:53,udp://180.184.1.1:53
   ttl: 0
   seq: top
   socks5: no
 - zone: vegslb.com
   dns: udp://223.5.5.5:53,udp://180.184.1.1:53
   ttl: 0
   seq: top
   socks5: no

**Comment by kkkgo** [comment]
这种简单的规则用RULES_TTL写会更方便。

**Comment by xiebaiyuan** [comment]
update my force_ttl_rules.txt   and RULES_TTL = 10 （magic）
douyinpic.com@180.184.1.1:53
douyincdn.com@180.184.1.1:53
amemv.com@180.184.1.1:53
douyinvod.com@180.184.1.1:53
douyinliving.com@180.184.1.1:53
douyinstatic.com@180.184.1.1:53
douyin.com@180.184.1.1:53
idouyinvod.com@180.184.1.1:53
byteimg.com@180.184.1.1:53
bytegecko.com@180.184.1.1:53

**Reply by Konbai-code** [reply]
感谢分享

**Reply by 742038744** [reply]
你好，除了这个方法，还找到其他好方案没？这个清单需不需要定期更新。

**Reply by xiebaiyuan** [reply]
> 你好，除了这个方法，还找到其他好方案没？这个清单需不需要定期更新。

没问题我就一直在用， 域名可能还好，基本不咋变。

**Reply by xiebaiyuan** [reply]
> 你好，除了这个方法，还找到其他好方案没？这个清单需不需要定期更新。

我还真找到了..  其实就是 geosite:bytedance

**Reply by gatalon** [reply]
> > 你好，除了这个方法，还找到其他好方案没？这个清单需不需要定期更新。
> 
> 我还真找到了.. 其实就是 geosite:bytedance

这个要怎么配置到force_ttl_rules.txt里面啊

**Reply by xingshu1990** [reply]
> > > 你好，除了这个方法，还找到其他好方案没？这个清单需不需要定期更新。
> > 
> > 
> > 我还真找到了.. 其实就是 geosite:bytedance
> 
> 这个要怎么配置到force_ttl_rules.txt里面啊

笨办法：手动复制https://github.com/v2fly/domain-list-community/blob/master/data/bytedance，
然后正则表达式：
.\*:.\* 匹配带有冒号的内容
#.*   匹配注释
然后正则表达式：(.*)
替换为\1@223.5.5.5:53 或者\1@114.114.114.114:53

然后用nano 或者vim打开force_ttl_rules.txt，把修改好的内容放进去，保存。

**Reply by mmm1h** [reply]
https://raw.githubusercontent.com/mmm1h/clashconfig/refs/heads/main/force_ttl_rules.txt
每天早上4点自动更新

**Comment by biggbuddy** [comment]
请问"刷抖音链接暴增问题"是指什么问题?

**Reply by xiebaiyuan** [reply]
https://github.com/kkkgo/PaoPaoDNS/discussions/123  这个问题。
当时有讨论，抖音似乎刷视频时有爆发式的dns请求，dns连接数爆炸增加。  
路由器差一点的就重启，  有的运营商会断联～

**Comment by jq88kazze** [comment]
刷快手也会连接数暴涨，有没有什么通用的解决方法呢
